Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/service/processor/AccessTokenRequestProcessor.kt

```diff
@@ -71,8 +69,11 @@ class AccessTokenRequestProcessor @Inject constructor(
         dccTicketingServer.getAccessToken(url, authorizationHeader, body, jwkSet)
     } catch (e: DccTicketingServerCertificateCheckException) {
         throw when (e.errorCode) {
-            CERT_PIN_NO_JWK_FOR_KID -> DccTicketingException.ErrorCode.ATR_CERT_PIN_NO_JWK_FOR_KID
-            CERT_PIN_MISMATCH -> DccTicketingException.ErrorCode.ATR_CERT_PIN_MISMATCH
+            DccTicketingServerCertificateCheckException.ErrorCode.CERT_PIN_HOST_MISMATCH,
```

Comment:
Importing the error codes would make it more readable, but that's only a minor thing

Reply:
I like it this way, because I see where immediatly where the errorCode comes from.
Also, the CI doesnt allow wildcard imports. So for every error we would have a single import which is ugly.
I would like to let is this way